docs/specs/devcontainer-features.md

```diff
@@ -199,13 +199,15 @@ Example:
 
 #### (2) The `installsAfter` Feature property
 
-This property is defined in an individual feature's `devcontainer-feature.json` file by the feature author.  `installsAfter` allows an author to provide the tooling hints on loose dependencies between Features.
+This property is defined in an individual feature's `devcontainer-feature.json` file by the feature author.  `installsAfter` allows an author to provide the tooling hints on loose dependencies between Features.  
+
+> This property is mostly useful for optimizing build time (by reordering the Feature installation to reduce installing a required CLI twice, for example).  Ideally, all Features should be able fully install themselve without requiring another Feature be pre-installed.
```

Comment:
```suggestion
> This property is mostly useful for optimizing build time (by reordering the Feature installation to reduce installing a required CLI twice, for example).  Ideally, all Features should be able to fully install themselves without requiring another Feature to be pre-installed.
```

This sentence sounded a lil off to me, attempted to change. (not sure if it's just me)